src/server/http/HTTPParser.cpp

```diff
@@ -3,10 +3,268 @@
 //
 
 #include "server/http/HTTPParser.hpp"
+#include "server/global/GlobalLogger.hpp"
+
+#include <sstream>
+
+namespace NotApache
+{
+	std::string methodAsString(const e_method& in) {
+		switch (in) {
+			case INVALID:
+				return "INVALID";
+			case GET:
+				return "GET";
+			case HEAD:
+				return "HEAD";
+			case POST:
+				return "POST";
+			case PUT:
+				return "PUT";
+			case DELETE:
+				return "DELETE";
+			case CONNECT:
+				return "CONNECT";
+			case OPTIONS:
+				return "OPTIONS";
+			case TRACE:
+				return "TRACE";
+		}
```

Comment:
Use a static const map for easy and fast searching. the main branch has a CreateMap() templated builder that you can use to make it. (one map for enum to string, another one for string to enum). then use map.find() to find your value.